kubernetes/helm_charts/datapipeline_jobs/values.j2

```diff
@@ -151,6 +151,7 @@ post-publish-processor:
 
     cloudstorage {
       metadata.replace_absolute_path={{ cloudstorage_replace_absolute_path | default('false') }}
+      relative_path_prefix={{ cloudstorage_relative_path_prefix_content }}
```

Comment:
@anilguptatarento Please add default value for the variable cloudstorage_relative_path_prefix_content

Reply:
Added the default in main.yml